Tests/SWBCoreTests/PIFLoadingTests.swift

```diff
@@ -683,24 +683,19 @@ private final class ProjectModelItemClass: ProjectModelItem {
         #expect(fileGroup.children.count == 2)
 
         // Examine its children
-        if let fileRef = fileGroup.children.first as? FileReference {
+        if let fileRef = try? #require(fileGroup.children.first as? FileReference) {
```

Comment:
I bet you can suppress the warning with:

```suggestion
        if let fileRef = try? #require(fileGroup.children.first as? FileReference?) {
```

Reply:
That works, and seems a simple way to suppress all but one of these warnings (the outlier isn't downcasting), while keeping the general approach. so I've incorporated it into the diffs here. Thanks!